src/components/text-pane/text-section.tsx

```diff
@@ -17,15 +18,20 @@ function TextSectionTitle({ count, title }: ITextSectionTitleProps) {
 
   const { actual, predicted, color } = title;
   return (
-    <span>
+    <span className="actual-title">
       {actual && (
-        <span>
-          True label: <span className="label" style={{ color }}>{actual}</span>
+        <>
+          <span>True label: </span><span className="label" style={{ color }}>{actual}</span>
           {predicted && <span>,&nbsp;</span>}
-        </span>
+        </>
       )}
-      {predicted && <span>Predicted label: <span className="label" style={{ color }}>{predicted}</span></span>}
-      <span>&nbsp;</span>{`(${count} case${count === 1 ? "" : "s"})`}
+      {predicted && (
+        <>
+          <span>Predicted label: </span><span className="label" style={{ color }}>{predicted}</span>
+        </>
```

Comment:
Seems like this could be simplified to:
```suggestion
        <span>Predicted label: </span><span className="label" style={{ color }}>{predicted}</span>
```

Reply:
I think it needs the `<></>` because it contains two `<span>`s next to each other. This whole area doesn't match the new design spec and I hope I get a chance to fix it up before StoryQ dev time winds down.